plugins/modules/ipagroup.py

```diff
@@ -377,6 +439,18 @@ def main():
                                              "group": group,
                                          }])
 
+                    if has_add_membermanager:
+                        # Add membermanager users and groups
+                        if len(membermanager_user) > 0 or \
+                           len(membermanager_group) > 0:
```

Comment:
Both `membermanager_user` and `membermanager_group` can be `None` here. It happened while running `test_group.yml` test set.

For example, this playbook fails on Fedora 32, FreeIPA 4.8.6:
```yaml 
  - name: Error due to membermanager being None
    ipagroup:
      ipaadmin_password: SomeADMINpassword
      name: group1
      group:
      - group2
      - group3
      action: member
    register: result
    failed_when: not result.changed
``` 

I'd suggest using `if membermanager_user or membermanager_group:` as it would cope with both `None` and empty lists.

Reply:
Testing for None is sufficient here.